### PR TITLE
Update virtualized list demo

### PR DESCRIPTION
**Pull request by rmarscher**
- Connect to trpc.car.all query
- Update web implementation to use tamagui stack wrapper
- Update web implementation to work with onScroll
- Add back button header with animation height on scroll

I also updated the database table names to lowercase plural to match the naming in the seed file. If you prefer, I could resubmit with keeping the table names and just updating the seed file.

Feel free to close if you want to go in a different direction. No worries. Thanks for all of the work on this project so far!

### CODE REVIEW

**Comment by timothymiller** [comment]
What's the proper casing of an SQL table?

I like `Car`, so the type inteferencing looks correct `Car[]` vs `Cars[]`

Is the first letter capitalized or is it all lower case?

**Comment by rmarscher** [comment]
I think it's subjective. Let me resubmit this preserving that naming. :)

**Comment by rmarscher** [comment]
OK @timothymiller. I reversed the table naming change and just corrected the table name in the seed file. Ready for review. The way this is solution calls the onScroll prop in the web version is a hack... but it works for tracking scroll position which I think is the main use case.

File: packages/app/utils/number.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function formatNumber(x: number): string {
+  return x.toLocaleString('en-US')
```
Comment (rmarscher) [inline packages/app/utils/number.ts:2]: I'm noticing that the locale would be a good var to be detected and stored in an atom like the theme...

**Pull request merged**